### PR TITLE
[PO-26] Add a dummy Election Manager

### PR DESCRIPTION
**Pull request by mirkoAlic**
The main goal of this abstraction is to verify and determine the stakeholder leader for a given slot number, given the epoch stake distribution, the stakeholder public key, and a specific slot in epoch.

ElectionManagerImpl
Like a first attempt, we propose to model a mock implementation for the Follow the Satoshi algorithm. It's just a round robin between each fixed stakeholder.

### Sub Tasks
Create a initial hardcoded stake distribution

### AC
a method called verify(stakholder public key, stake distribution, slot in epoch), return true or false.

### Assumptions
Only a fixed group of stakeholders will be able to add new blocks(mine blocks).
The group of stakeholders will not be updated from epoch to epoch.

### CODE REVIEW

File: src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala [hunk 1]
```diff
@@ -77,6 +78,15 @@ trait DiscoveryConfigBuilder {
   lazy val discoveryConfig = DiscoveryConfig(Config.config)
 }
 
+trait OuroborosConfigBuilder {
+  lazy val ouroborosConfig = OuroborosConfig(Config.config)
+}
+
+trait ElectionManagerImplBuilder {
```
Comment (ntallar) [inline src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala]: Minor detail: Should this maybe be an `ElectionManagerBuilder`? So then here we would create it's instance from the chosen implementation.

File: src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala [hunk 2]
```diff
@@ -478,3 +488,4 @@ trait Node extends NodeKeyBuilder
   with KnownNodesManagerBuilder
   with SyncConfigBuilder
   with MinerBuilder
+  with OuroborosConfigBuilder
```
Comment (ntallar) [inline src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala:491]: Should this also extend the `ElectionManagerBuilder`?

File: src/universal/conf/ouroboros.conf [hunk 3]
```diff
@@ -0,0 +1,9 @@
+mantis {
+
+  ouroboros {
+
+    known-stakeholders = [
```
Comment (ntallar) [inline src/universal/conf/ouroboros.conf]: Very minor detail: Should this configuration be commented out as done with other configurations in this `config` package? We probably have to move the comment on it (from the `application.conf`) for anyone configuring it, right?

**Review by ntallar — COMMENTED** [review]
Apart from those minor comments, LGTM!

File: src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala (hunk 2 above)
Comment (mirkoAlic) [reply src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala:491]: :+1:

File: src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala (hunk 1 above)
Comment (mirkoAlic) [reply src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala]: :+1:

**Review by ntallar — APPROVED** [review]

**Pull request merged**